scripts/app

```diff
@@ -106,6 +114,14 @@ update_installed_apps() {
   echo "${updated_json}" > "${USER_FILE}"
 }
 
+# Make sure no other install script is running, if it is, wait for it to exit
+if [[ "$(check_if_not_already_running)" == "2" ]]; then
```

Comment:
Rather than halting the execution of the entire script, which prevents apps from being installed in parallel, we could maybe check if the JSON file is currently being updated with a write lock, which allows apps to be installed/uninstalled in parallel and just briefly blocked while waiting to do the write.